test_e2e.sh

```diff
@@ -13,5 +13,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+CHECK_DOCS_ONLY=$("test/check_docs_only.sh")
+if [ -z "$CHECK_DOCS_ONLY" ]; then
+    ./test/e2e/ci.sh
+    exit 0
+fi
 
```

Comment:
Also, we need to log ( echo ) something like WARNING: The tests were skipped because only changes on the docs were faced.